Tasks/AzureWebAppContainerV1/package.json

```diff
@@ -21,7 +21,10 @@
     "@types/node": "^10.17.0",
     "@types/q": "1.0.7",
     "@types/uuid": "^8.3.0",
-    "azure-pipelines-tasks-azurermdeploycommon": "^3.221.1",
+    "azure-devops-node-api": "11.2.0",
+    "azure-pipelines-task-lib": "^4.1.0",
```

Comment:
Task lib 3.4.0 might be better since task has only Node10 version.

Reply:
Done